README.md

```diff
@@ -1 +1,88 @@
-# leaderboard
\ No newline at end of file
+# Leaderboard
+
+> The leaderboard website displays scores submitted by different players. It also allows you to submit your score.
+
+
+## Built With
+
+- HTML, CSS, JavaScript
+- Webpack
+- Git & GitHub
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Get Started
+
+To get the content of this project locally you need to run the following commands in your terminal.
+
+### Prerequisites
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone https://github.com/Samuellekpor/leaderboard.git
+```
+3. To access cloned directory run:
+```bash
+cd MyPortfolio
+```
+3. Open it with the live server
```

Comment:
- [ ] Kindly add `npm run start` here to help the viewer to start the project correctly.